core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -601,6 +601,7 @@ context, new FileSystemJournalEntryMerger()
   @Override
   public void start(Boolean isPrimary) throws IOException {
     super.start(isPrimary);
+    long startSt = System.currentTimeMillis();
```

Comment:
Could you use a stopwatch object here instead please?

Reply:
done